api/app/wildfire_one.py

```diff
@@ -481,6 +530,51 @@ async def get_hourly_actuals_all_stations(
     return hourly_actuals
 
 
+async def get_wfwx_stations_from_station_codes(session, header, station_codes: Optional[List[int]]):
+    """ Return the WFWX station ids from WFWX API given a list of station codes. """
+    fire_centre_station_codes = await get_fire_centre_station_codes()
```

Comment:
get_fire_centre_station_codes doesn't do anything async, so you can fix it's definition and remove the await.

Reply:
you can also move this section to the `if station_codes is None:` block - `since fire_centre_station_codes` is only used there.